stats/bonuses/lucarioAuraBerserk.lua

```diff
@@ -51,7 +54,7 @@ function update(dt)
 		self.critMod=0
 	end
 		-- make sure it doesn't get wonky by setting limits
-    if (self.powerMod) < 1 then
+    if (self.powerMod) <= 1 then
```

Comment:
There's no need for "<=" here. The check here should be activated only when "self.powerMod" is less than 1, or the script will go into infinite loop.

Reply:
Thanks for catching that!